### DISCUSSION TITLE
Define no-go areas

### DISCUSSION THREAD

**Discussion by Leeb64** [devemux86/cruiser#537] [Q&A] [answered]
NoGo Aereas are a very practical thing - but a bit complicated to define.
If you could add a feature for NoGoAerias, similar to Favorites?
Select a NoGoAeria with crosshair, add radius,name.
Saved in “my collection”.

**Comment by devemux86** [comment, marked as answer]
@Leeb64 thanks for the suggestion.

Let's discuss it in the topic I had already prepared:
- https://github.com/devemux86/cruiser/discussions/538
